src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray.cs

```diff
@@ -106,14 +106,18 @@ public static ImmutableArray<T> CreateRange<T>(IEnumerable<T> items)
             var immutableArray = items as IImmutableArray;
             if (immutableArray != null)
             {
-                immutableArray.ThrowInvalidOperationIfNotInitialized();
+                Array array = immutableArray.Array;
+                if (array == null)
+                {
+                    throw new InvalidOperationException(SR.InvalidOperationOnDefaultArray);
+                }
 
                 // immutableArray.Array must not be null at this point, and we know it's an
```

Comment:
Do you want to update the comment so it discusses the `array` variable instead of `immutableArray.Array`, now that the code beneath it deals with `array`?